.github/workflows/smartbear-issue-comment-created.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   jira:
     runs-on: ubuntu-latest
-    if: ${{ github.event.comment.body == '/jira ticket' }}
+    if: ${{ github.event.comment.body == '/smartbear-supported ticket' }}
```

Comment:
Sorry, extra thought, given they're all called "issues", what about `/smartbear-supported issue`

Reply:
yep, I like it even better